lib/models/blueprint.js

```diff
@@ -850,7 +850,9 @@ Blueprint.prototype._locals = function(options) {
   var sanitizedModuleName = moduleName.replace(/\//g, '-');
 
   return new Promise(function(resolve) {
-    resolve(this.locals(options));
+    return resolve(this._processPrompts(options));
+  }.bind(this)).then(function (prompts) {
```

Comment:
I feel like that's a hacky way to do it, almost an unnecessary step just to not use `new`.